src/sfizz/modulations/ModKey.cpp

```diff
@@ -87,6 +87,18 @@ std::string ModKey::toString() const
         return absl::StrCat("Pitch {", region_.number(), "}");
     case ModId::Volume:
         return absl::StrCat("Volume {", region_.number(), "}");
+    case ModId::FilGain:
+        return absl::StrCat("FilterGain {", region_.number(), ", N=", params_.N, "}");
```

Comment:
The displayed filter and EQ number should be from 1.
It's suggested to increment, same as envelope and LFO.

Reply:
I think I changed it in the next PR.

Reply:
Yes I did in 83cd4b631796875cddf5d6c3a358bdf84eaeaea6